app/views/layouts/govuk_admin_template.html.erb

```diff
@@ -121,15 +122,17 @@
 
           ga('create', 'UA-26179049-6', '<%= ENV.fetch('GOVUK_APP_DOMAIN') %>');
           ga('set', 'anonymizeIp', true);
+          <%= content_for(:before_pageview_js) %>
           <% if content_for?(:custom_pageview_fullpath) %>
             GOVUKAdmin.trackPageview('<%= content_for(:custom_pageview_fullpath) %>');
           <% else %>
             ga('send', 'pageview');
           <% end %>
         </script>
-      <% elsif Rails.env.development? %>
+      <% elsif allow_google_analytics_stub %>
         <script>
           if (console) {window.ga = function() {console.log.apply(console, arguments);};}
+          <%= content_for(:before_pageview_js) %>
```

Comment:
An example of using this to test GA in a publisher can be seen in: https://github.com/alphagov/signon/pull/557 (in particular this helper file: https://github.com/alphagov/signon/pull/557/files#diff-8cb67762c93c9433824b186e8a460a6c)